lib/gems/pending/appliance_console/scap.rb

```diff
@@ -20,7 +22,7 @@ def lockdown
     private
 
     def yaml_filename
-      File.expand_path("config/scap_rules.yml", __dir__)
+      File.expand_path("productization/appliance_console/config/scap_rules.yml", RAILS_ROOT)
```

Comment:
I think this would be better if we just passed the config file or directory in `initialize`; that way we could avoid spreading the knowledge of where `RAILS_ROOT` may be.